Lib/fontTools/cffLib.py

```diff
@@ -1731,17 +1743,16 @@ def xmlRead(self, name, attrs, content, parent):
 	((12, 36),	'FDArray',		'number',	None,	FDArrayConverter()),
 	(17,		'CharStrings',		'number',	None,	CharStringsConverter()),
 	(24,		'VarStore',		'number',	None,	VarStoreConverter()),
-	(25,		'maxstack',		'number',	None,	None),
```

Comment:
I wonder, if this `maxstack` is a CFF2-only operator why is it also listed in `topDictOperators` and not just in `topDictOperatos2`?
I see it's mentioned in CFF2 spec https://www.microsoft.com/typography/otspec/cff2charstr.htm, but don't see any reference to it in the CFF 1.0 spec (Tech Note #5176).